marimo/_runtime/runtime.py

```diff
@@ -215,6 +215,28 @@ def query_params() -> QueryParams:
     return get_context().kernel.query_params
 
 
+@mddoc
+def cli_args() -> CLIArgs:
+    """Get the command line arguments of a marimo app.
+
+    **Examples**:
+
+    ```python3
+    # Access the command line arguments
+    size = mo.cli_args().get("size") or 100
+
+    for i in range(size):
+        print(i)
+    ```
+
+    **Returns**:
+
+    - A dictionary containing the command line arguments.
+      This dictionary is read-only and cannot be mutated.
+    """
+    return get_context().kernel.cli_args
```

Comment:
This will throw when run as a script.

Catch `ContextNotInitializedError` and warn that CLI args not yet supported when running as a script?